src/components/Loader/Loader.scss

```diff
@@ -0,0 +1,25 @@
+.Loader {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+
+  &__content {
+    border-radius: 50%;
+    width: 2em;
+    height: 2em;
+    margin: 1em auto;
+    border: 0.3em solid #ddd;
+    border-left-color: #000;
+    animation: load8 1.2s infinite linear;
+  }
+}
+
+@keyframes load8 {
```

Comment:
It would be helpful to add a comment above the '@keyframes load8' explaining the purpose of this animation. This can improve code readability. 📚